courses/utils.py

```diff
@@ -89,3 +89,16 @@ def get_year_season_from_course_run(course_run):
         return season_year_tuple[0], NUMBER_SEASON_MAP[season_year_tuple[1]]
     else:
         return season_year_tuple
+
+
+def is_blank(string):
```

Comment:
please do not use reserved keywords (`string` is a module of the standard library)